CONTRIBUTING.md

```diff
@@ -1,177 +1,141 @@
 # Contributing to Operation Code
 
-Looking to contribute something to Operation Code? **Here's how you can help.**
+So you want to learn how to program? Contributing to Operation Code is a great place to get started! This document will help you get from 0 to deploying code in no time! 
 
```

Comment:
`0 to deploying` reads strangely to me ... more easily read as `zero to deploying`